stable/insights-agent/templates/install-reporter/job.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  {{- include "metadata" "install-reporter" }}
+  annotations:
+    "helm.sh/hook": pre-install,pre-upgrade
+    "helm.sh/hook-delete-policy": hook-succeeded,before-hook-creation
+spec:
+  ttlSecondsAfterFinished: 30
+  backoffLimit: {{ .Values.cronjobs.backoffLimit }}
+  template:
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: start-job-from-cronjob
+        image: "{{ .Values.installReporter.image.repository }}:{{ .Values.installReporter.image.tag }}"
+        imagePullPolicy: Always
+        command: ["curl"]
+        args:
+          - "-X"
+          - "POST"
+          - "{{ .Values.insights.host }}/v0/organizations/{{ .Values.insights.organization }}/clusters/{{ .Values.insights.cluster }}/data/installation"
+          - "-H"
+          - "Content-Type: application/json"
+          - "-H"
+          - "Authorization: Bearer {{ .Values.insights.base64token | b64dec }}"
```

Comment:
We should probably mount the secret as an environment variable. This would expose the token in the YAML.

Reply:
Right you are.